web-ui/src/main/resources/catalog/locales/nl-core.json

```diff
@@ -380,5 +380,6 @@
     "resolveCaptcha": "Captcha oplossen",
     "resolutions": "Resolutie",
     "loginTitle": "Inloggen",
-    "loginText": "Login met uw gebruikersnaam en wachtwoord om aan de catalogus bij te dragen."
```

Comment:
Should be only committed changes to english files, the other languages should be translated in Transifex. Please remove this file from the pull request, thanks.